### PR TITLE
Added cal.com banners. Fixed incomplete sentence

### PR DESCRIPTION
**Pull request by DaveFlynn**
**PR checklist**

- [ ] Ensure you have added or ran the appropriate tests for your PR.
- [ ] DCO signed

**What type of PR is this?**

Update ReadMe.

**What this PR does / why we need it**:

Add cal.com links per sponsorship criteria.

**Which issue(s) this PR fixes**:

N/A

**Special notes for your reviewer**:

Only readme text updated.

**Does this PR introduce a user-facing change?**:

### CODE REVIEW

File: README.md
Comment (DaveFlynn) [inline README.md:1]: regarding line #2 - I found `target="_blank"` doesn't work, so removed it

Comment (wcchang1115) [reply README.md:1]: Thanks!
I found a [stack overflow post](https://stackoverflow.com/questions/41915571/open-link-in-new-tab-with-github-markdown-using-target-blank) that also answered this.
It's no way to make it.

**Review by wcchang1115 — APPROVED** [review]
All good, thanks!

File: README.md
Comment (DaveFlynn) [reply README.md:1]: Thanks for checking. I found it's a pain to preview the markdown before merging, so difficult to know if it'll work or not. Good to see it's not me!

**Pull request merged**